src/boosting/gbdt.cpp

```diff
@@ -12,7 +12,7 @@
 #include <chrono>
 #include <string>
 #include <vector>
-
+#include <queue>
```

Comment:
Remove this.  Maybe you want to use queue to optimize the storage of historical stores. But add this when you finished.